examples/mobile/HomeApp/js/app.js

```diff
@@ -125,6 +125,24 @@ import Storage from "./clipping-storage.js";
 				(app2.isActive) ? 0 : -1 : 1
 		});
 
+		// check webclips order in app
+		appsList = appsList.forEach((app) => {
+			fetch(`${app.webClipsList[0].url}/manifest.json`)
```

Comment:
How often the manifest will change? Is this possible during the app runtime? I think will be better to read the manifest if not exists in apps list yet and store in apps list for next use. This reduce number of reading.